client/state/form/reducer.js

```diff
@@ -21,8 +22,12 @@ reducers[ SET_FORM_PROPERTY ] = ( state, action ) => {
 	return Object.assign( {}, state, newObj );
 };
 
-reducers[ NEXT_STEP ] = ( state ) => {
-	return Object.assign( {}, state, { currentStep: state.currentStep + 1 } );
+reducers[ BACK_FROM_SUGGESTION ] = ( state ) => {
+	return [
+		setFormProperty( 'errors', {} ),
+		setFormProperty( 'acceptSuggestion', undefined ),
+		setFormProperty( 'pristine', false ),
+	].reduce( form, state );
```

Comment:
I implemented it this way on purpose to trigger a discussion. What do you think about this way of implementing an action which must perform a sequence of tasks? The other way to do it would be to make an action creator with several `dispatch()` calls. I can see pros and cons either way. We want to keep reducers simple, but if we go too far then the Redux log will just be plagued with `SET_PROPERTY`, `UPDATE_FIELD` actions that are _not_ semantic at all.